snapshot/bundlestore/groupcache.go

```diff
@@ -0,0 +1,93 @@
+package bundlestore
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/golang/groupcache"
+	"github.com/scootdev/scoot/cloud/cluster"
+)
+
+type PeerFetcher interface {
+	Fetch() ([]cluster.Node, error)
+}
+
+type GroupcacheConfig struct {
```

Comment:
comments/docs somewhere would be helpful....need to understand the functionality this code is trying to provide, and what will be using it.

I'm assuming this functionality is to support caching groups of ?snapshots? ?
What is the grouping criteria - where is a group defined?

This GroupcacheConfig the configuration for a group or an entry in the group?
- is Name the name of the Cache that contains a group of things/?snapshots? or the name of one of the things in the group?
- is Memory_btyes - does this imply that the cache contents are stored in memory? then what is AddrSelf and Endpoint?
- are peers other groupCaches or other entries in the cache that this entry is in?